Yoast/wordpress-seo
# Update snippet variable tab of the help center to have a labels column

## Summary

This PR can be summarized in the following changelog entry:

* Updated the snippet variables tab of the help center to have the new names.

## Relevant technical choices:

* Changed the value of the array to an object with the new label, and the description.
* Added all niceNames from defaultReplaceVariables.
* All external plugins registering help texts for their snippet variables get the name as the label for now. This should be updated in the future. 

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch, do composer install.
* Open a page/post. (or any other place with the snippet variables help tab, such as search appearance)
* Visit the help center.
* Verify the labels column is present.

## Quality assurance


Fixes #10054
